src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -54,33 +54,58 @@ public void storeMessage(Message message) {
    *     message. List is sorted by time descending.
    */
   public List<Message> getMessages(String user) {
-    List<Message> messages = new ArrayList<>();
-
     Query query =
         new Query("Message")
             .setFilter(new Query.FilterPredicate("user", FilterOperator.EQUAL, user))
             .addSort("timestamp", SortDirection.DESCENDING);
-    PreparedQuery results = datastore.prepare(query);
-
+    List<Message> messages = fetchMessages(query);
+    
+    return messages;
+  }
+  
+  /**
+   * Gets messages posted by all users.
+   *
+   * @return a list of messages posted by all users, or an empty list if no user has posted a 
+   *     message. List is sorted by time descending.
```

Comment:
Formatting is weird here, why is a on its own line? Might just be code review wrapping though

Reply:
Yeah I am using Eclipse and it is doing funky things by default. I will go fix now

Reply:
What do you mean 'a in its own line' ? I don't see that on my screen .PS formatting is fixed now!